extra/utils.py

```diff
@@ -74,8 +74,9 @@ def find_class(self, module, name):
         elif name == "_rebuild_parameter":
           return HackParameter
       else:
+        if module.startswith('pytorch_lightning'): return Dummy
```

Comment:
also `pytorch_lightning` enables this fancy traceback printer, so maybe better to skip importing it completely?

https://rich.readthedocs.io/en/stable/traceback.html